jzp-code/kukdo/project/cloud-storage/cloud-storage-parent/cloud-storage-web/src/main/java/com/xiaomi/model/FileMetadata.java

```diff
@@ -0,0 +1,51 @@
+package com.xiaomi.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+import java.time.LocalDateTime;
+
+@Entity
+public class FileMetadata {

```

Comment:
in mvc, the "model" represents the data and the business logic of the application